### PR TITLE
feat(starr): Update of the French Guide

### PR DESCRIPTION
**Pull request by NiceTSY**
# Pull Request

## Purpose

Updating the French Guide to include improvement from latest Radarr/Sonarr changes, to improve upon specific settings, and streamline some approach.

## Open Questions and Pre-Merge TODOs

- [x] Implement new Language CFs to align with the rest of the guide language recommendation
- [x] Update the Radarr French Guide
- [x] Update the Sonarr French Guide
- [x] Update some French CFs
- [ ] (Maybe) Update the Sonarr French Anime Guide
- [x] Check language errors again
- [x] Check technical errors again
- [x] Check scoring a last time

## Requirements

- [x] These changes meet the standards for [contributing](https://github.com/TRaSH-Guides/Guides/blob/master/CONTRIBUTING.md).
- [x] I have read the [code of conduct](https://github.com/TRaSH-Guides/Guides/blob/master/.github/CODE_OF_CONDUCT.md).

### CODE REVIEW

**Review by TRaSH- — CHANGES_REQUESTED** [review]

File: includes/french-guide/radarr-cf-french-hd-bluray-web-scoring-en.md
Comment (NiceTSY) [reply includes/french-guide/radarr-cf-french-hd-bluray-web-scoring-en.md:1]: Thanks, I tried to put everything under `french-xxx` but missed some apparently will check and update those.

**Review by TRaSH- — DISMISSED** [review]

**Comment by TRaSH-** [comment]
Added do not merge because there are still a view things that might need to be updated

**Comment by TRaSH-** [comment]
Do you still want to update the anime guide, or is that something for another time?

**Comment by TRaSH-** [comment]
I've got a green light to merge this in our dev channel. The anime guide will be updated later.

**Comment by LhuanDula** [comment]
Hi,
I noticed that the multi and multi-french trashids format have been deleted but they are still mentioned in the guide (both in english and french).
Are we supposed to drop the use of the multi-french custom format (which is a shame because that's the one that has really worked for me)?
Cheers,

**Pull request merged**
